compose.freeswitch.yml

```diff
@@ -0,0 +1,19 @@
+services:
+  freeswitch:
+    image: docker.io/rocketchat/freeswitch:latest
+    restart: always
+    environment:
+      EXTERNAL_ADDRESS_FQDN: ${VOIP_DOMAIN:?}
+      FREESWITCH_PASSWORD: ${FREESWITCH_PASSWORD:-}
+      RTP_START_PORT: ${VOIP_RTP_START_PORT:-20000}
+      RTP_END_PORT: ${VOIP_RTP_END_PORT:-20046}
+      ACCEPT_EXTERNAL_CALLS: false
+      USE_LEGACY_ARCH: false
```

Comment:
Can we add the explanation for these variables as well, (As they are added for compose.freeswitch.external.ym.?